sporks.json

```diff
@@ -0,0 +1,950 @@
+{
```

Comment:
do we want both then? past-sporks.mdx and this file?

Reply:
We could use this file to populate the docs pages. That might be handy.

Reply:
Good question Vishal. I guess it depends if we think it's best to share all the details in both places, or if we're keeping the .mdx file a subset.

Reply:
I like the idea of using the json to generate the mdx file